fractal_server/string_tools.py

```diff
@@ -43,3 +48,23 @@ def slugify_task_name_for_source(task_name: str) -> str:
         Slug-ified task name.
     """
     return task_name.replace(" ", "_").lower()
+
+
+def validate_cmd(command: str, allow_char: Optional[str] = None):
+    """
+    Assert that the provided `command` does not contain any of the forbidden
+    characters for commands
+    (fractal_server.string_tools.__NOT_ALLOWED_FOR_COMMANDS__)
+
+    Args:
+        command: command to validate.
+        allow_char: chars to accept among the forbidden ones
+    """
+    forbidden = set(__NOT_ALLOWED_FOR_COMMANDS__)
+    if allow_char is not None:
+        forbidden -= set(allow_char)
```

Comment:
How much faster is this (if it is faster at all)?

My guess is that it makes no relevant difference, and then I suggest we keep it more readable (`forbidden = forbidden - set(allow_char)`